.dockerignore

```diff
@@ -0,0 +1,4 @@
+Dockerfile
+README.md
+node_modules
```

Comment:
Shouldn't there be a slash at the end there?

Reply:
do not think so

Reply:
`node_modules` is a directory, and we mark directories as `node_modules/`. Unless this breaks the ignore rules, we should use it. Just like in `.gitignore`. Also, let's sort the list (again, like `.gitignore`).

Reply:
https://www.geeksforgeeks.org/how-to-use-a-dockerignore-file/

look on step 2